TEST_APPS/device/socket_app/cmd_socket.cpp

```diff
@@ -79,29 +81,60 @@
                             " <id> join_bg_traffic_thread\r\n"\
                             " <id> setsockopt_keepalive <seconds[0-7200]>\r\n"\
                             " <id> getsockopt_keepalive\r\n"\
-                            "\r\nFor TCPServer\r\n"\
                             " <id> listen [backlog]\r\n"\
+                            " <id> accept\r\n" \
+                            "   accept new connection and returns new socket ID\r\n" \
+                            "\r\nFor TCPServer\r\n"\
                             " <id> accept <new_id>\r\n"\
                             "   accept new connection into <new_id> socket. Requires <new_id> to be pre-allocated.\r\n"\
                             "\r\nOther options\r\n"\
                             " print-mode [--string|--hex|--disabled] [--col-width <width>]"
 
-class SInfo;
-static Queue<SInfo, 10> event_queue;
-static int id_count = 0;
+
+
```

Comment:
A minor nit, but won't let it block the PR.

Might be useful to leave a comment as to what this cert directs to.